src/common/store/job/actions.js

```diff
@@ -98,8 +97,10 @@ const uploadPdfFile = createStep('FILE_UPLOAD', 30, async (dispatch, getState) =
         dispatch(updatePdfFile(fileDescriptor));
     } else {
         const link = getFileLink(getState());
-        uploadLinkAction(link);
         const fileDescriptor = await FileService.uploadLink(link);
+        if (fileDescriptor.contentType !== 'application/pdf') {
+            throw new Error('Content type is not application/pdf');
```

Comment:
Let's replace the error message with something like that "Unsupported content type: ${fileDescriptor.contentType}"